devops/windows/base-docker-wsl/docker-stack.yml

```diff
@@ -0,0 +1,268 @@
+version: '3.8'
+
+networks:
+  backend:
+    name: backend
+    driver: overlay
+    attachable: true
+  frontend:
+    name: frontend
+    driver: overlay
+    attachable: true
+
+services:
+
+  postgresql:
+    image: skunkworks2:5000/v2/bitnami/postgresql:15.4.0
+    networks:
+    - backend
+    ports:
+    - published: 5432
+      target: 5432
+      protocol: tcp
+      mode: host
+    volumes:
+    - type: bind
+      source: /mnt/c/Users/skunkworks/Projects/jempi/JeMPI/devops/windows/base-docker-wsl/docker_data/data/postgres
+      target: /conf
+      read_only: true
+    environment:
+      POSTGRESQL_USERNAME: postgres
+      POSTGRESQL_PASSWORD: postgres
+      POSTGRESQL_DATABASE: notifications
+      ALLOW_EMPTY_PASSWORD: "no"
+    deploy:
+      mode: replicated
+      replicas: 0
+      resources:
+        limits:
+          memory: 8G
+      update_config:
+        parallelism: 1
+        delay: 10s
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+        max_attempts: 0
+      placement:
+        constraints:
+        - node.labels.name == skunkworks2
```

Comment:
Should this not be equals to an external variable (i.e `${PLACEMENT_POSTGRESQL}`)

Reply:
p.s this is also the same for all other services below

Reply:
same as a previous response --- it get created from a template using 
  envsubst < ./conf/stack/docker-stack-${SPEC_SETTINGS}.yml > ./docker-stack.yml